session4/src/com/itschool/homework/Challenge7.java

```diff
@@ -0,0 +1,16 @@
+package com.itschool.homework;
+
+public class Challenge7 {
+
+    public static void main(String[] args) {
+
+        int a = 10;
+        double b = 10.5;
+
+        System.out.println(a+b);
+        System.out.println(a*b);
+        System.out.println(a-b);
+        System.out.println(a/b);
+        System.out.println(a%b);
```

Comment:
this is great, but please format the code before committing it (In IntelliJ hamburger menu (top left) -> Code -> reformat code. Or, if you are using Eclipse Keymap press CTRL + Shift + F).